libs/service-portal/documents/src/lib/types.ts

```diff
@@ -11,3 +11,15 @@ export type ActiveDocumentType = {
   categoryId?: string
   senderNatReg?: string
 }
+
+export type ActiveDocumentType2 = {
+  document: Partial<DocumentV2Content>
+  id: string
+  subject: string
+  date: string
+  sender: string
+  downloadUrl: string
+  img?: string
+  categoryId?: string
+  senderNatReg?: string
+}
```

Comment:
Why the copy-paste instead of just `document: DocumentDetails | Partial<DocumentV2Content>`, and some type discrimination like, `document instanceof DocumentDetails`